apps/client/src/shared/mocks/top-carousel-mock.ts

```diff
@@ -0,0 +1,61 @@
+export const performData = [
```

Comment:
```suggestion
export const PERFORMANCE_DATA = [
```
SNAKE_CASE로 변경 부탁드려요 ~
그리고 data로 depth 하나 더 깊게 !! 명세서대로 부탁드립니다 ~

Reply:
React Query의 select 옵션을 사용하여 필요한 데이터만 반환하도록 할 예정이라 depth를 깊게 하지 않아도 될 것 같다고 생각합니다~! 어떻게 생각하시나요 !!